### PR TITLE
feat: add code magic

### PR DESCRIPTION
**Pull request by Dun-sin**
[CODE-MAGIC](code-magic.vercel.app/)

Code Magic is a free tool that allows you to generate Tailwind and css code

### CODE REVIEW

**Comment by markodenic** [comment]
@Dun-sin The link is broken. Can you please adjust it?

File: README.md [hunk 1]
```diff
@@ -555,6 +555,7 @@ Initially created by [Marko](https://markodenic.com) at [Web Development Resourc
 | [Toptal CSS3 Generator](https://www.toptal.com/developers/css3maker)                  | A free online tool for quickly generating CSS3 snippets, such as for effects, gradients and animations.                                                                               |
 | [Neumorphism](https://neumorphism.io/)                                                | A free online tool for designing attractive UI with colors, gradients and shadows..                                                                                                   |
 | [Scrollbar.app](https://scrollbar.app/)                                               | A simple online tool for creating custom scrollbars for the web.                                                                                                                      |
+| [Code Magic](Code-magic.vercel.app)                                                   | A free tool to make CSS easier by generating Tailwind and CSS code for effects, gradients and inputs |
```
Comment (Dun-sin) [inline README.md:558]:
```suggestion
| [Code Magic](https://code-magic.vercel.app/)                                                   | A free tool to make CSS easier by generating Tailwind and CSS code for effects, gradients and inputs |
```

**Comment by Dun-sin** [comment]
> @Dun-sin The link is broken. Can you please adjust it?

@markodenic done

**Review by markodenic — APPROVED** [review]
LGTM!

**Pull request merged**